### PR TITLE
Add button to manually trigger a Random

### PR DESCRIPTION
**Pull request by Shadowrs**
Same PR as #251 but without the unrelated commits pushed by mistake

### CODE REVIEW

File: src/main/java/org/parabot/core/ui/RandomUI.java [hunk 1]
```diff
@@ -61,20 +68,31 @@ public void openFrame(ArrayList<String> randoms) {
...
     public void actionPerformed(ActionEvent e) {
-        Context.getInstance().getRandomHandler().clearActiveRandoms();
-        if (checkBoxes != null && checkBoxes.size() > 0) {
-            for (JCheckBox checkBox : this.checkBoxes) {
-                if (checkBox.isSelected()) {
-                    for (Random r : Context.getInstance().getRandomHandler().getRandoms()) {
-                        if (r.getName().equalsIgnoreCase(checkBox.getText().toLowerCase())) {
-                            Core.verbose("Actived random '" + r.getName() + "'");
-                            Context.getInstance().getRandomHandler().setActive(r.getName());
+        org.parabot.api.output.Logger.debug("RandomUI", e.toString()+" | "+e.getActionCommand());
```
Comment (JKetelaar) [inline src/main/java/org/parabot/core/ui/RandomUI.java]: Can the concatenation have spaces?

File: src/main/java/org/parabot/core/ui/RandomUI.java [hunk 2]
```diff
@@ -61,20 +68,31 @@ public void openFrame(ArrayList<String> randoms) {
...
+        if (e.getActionCommand().startsWith("Run now: ")) {
+            org.parabot.api.output.Logger.debug("RandomUI", e.getActionCommand().substring(9));
+            for (Random r : Context.getInstance().getRandomHandler().getRandoms()) {
+                if (r.getName().equalsIgnoreCase(e.getActionCommand().substring(9))) {
```
Comment (JKetelaar) [inline src/main/java/org/parabot/core/ui/RandomUI.java]: This substring isn't totally clear to me, could you elaborate this?

Comment (Shadowrs) [reply src/main/java/org/parabot/core/ui/RandomUI.java]: Agreed basing the link to a random on text alone is weak. We could attach the linked randoms' class via usage of `clientProperty` which is just a small hashmap (key:random, value:class). Not exactly what it was designed for but small footprint and reduced code of creating a map of button:random ourselves.

**Comment by Shadowrs** [comment]
Revamped 

alt

File: src/main/java/org/parabot/core/ui/RandomUI.java [hunk 3]
```diff
@@ -39,62 +39,61 @@ public void openFrame(ArrayList<String> randoms) {
...
             JLabel lblNone = new JLabel("None (yet).");
             lblNone.setBounds(6, 35, 120, 16);
             frame.getContentPane().add(lblNone);
         }
 
-        btnSubmit.addActionListener(this);
+        btnSubmit.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                Logger.addMessage("Active randoms count: "+Context.getInstance().getRandomHandler().getActiveRandoms().size());
```
Comment (Shadowrs) [inline src/main/java/org/parabot/core/ui/RandomUI.java]: Added spaces in next commit

**Pull request closed without merging**